lib/data_distributor/cpr/person.rb

```diff
@@ -0,0 +1,83 @@
+module DataDistributor
+  module CPR
+    #This class represents a person as known by cpr
+    class Person
+      attr_reader :data, :cpr
+
+      # @param [Dataset]
```

Comment:
```suggestion
      # @param [Hash]
```

Er det ikke et Hash den får?